lib/DxilPIXPasses/DxilShaderAccessTracking.cpp

```diff
@@ -244,7 +244,7 @@ void DxilShaderAccessTracking::EmitAccess(LLVMContext & Ctx, OP *HlslOP, IRBuild
 
   (void)Builder.CreateCall(AtomicOpFunc, {
       AtomicBinOpcode,// i32, ; opcode
-      m_HandleForUAV, // %dx.types.Handle, ; resource handle
+      m_FunctionToUAVHandle[Builder.GetInsertBlock()->getParent()], // %dx.types.Handle, ; resource handle
```

Comment:
Prefer `.at` so we get an exception if the lookup fails rather than null.

Reply:
Fair point. Thanks.